pushy/src/main/java/com/relayrides/pushy/apns/util/SimpleApnsPushNotification.java

```diff
@@ -217,6 +234,13 @@ public boolean equals(final Object obj) {
         } else if (!this.topic.equals(other.topic)) {
             return false;
         }
+        if (this.collapseId == null) {
+            if (other.collapseId != null) {
+                return false;
+            }
+        } else if (!this.collapseId.equals(other.collapseId)) {
+            return false;
+        }
```

Comment:
@123avi I mean the new method available since 1.7 which works exactly as the code above.